src/main/java/com/redhat/rhjmc/containerjfr/net/WebServer.java

```diff
@@ -234,14 +233,14 @@ private Response serveClientIndex() {
 
         @SuppressFBWarnings("DMI_HARDCODED_ABSOLUTE_FILENAME")
         private Response serveClientAsset(String assetName) {
-            Path assetPath = Paths.get("/", "web-client", assetName);
-            if (!assetPath.toFile().isFile()) {
+            URL url = WebServer.class.getResource(assetName);
+            if (url == null) {
                 return newFixedLengthResponse(Status.NOT_FOUND, NanoHTTPD.MIME_PLAINTEXT,
                         String.format("%s not found", assetName));
             }
             try {
-                String mime = NanoHTTPD.getMimeTypeForFile(assetPath.toUri().toString());
-                InputStream assetStream = Files.newInputStream(assetPath);
+                String mime = NanoHTTPD.getMimeTypeForFile(url.toString());
+                InputStream assetStream = url.openStream();
```

Comment:
Then if the above works, we can forego the `url` altogether and use `WebServer.class.getResourceAsStream` and skip this `openStream` step.

Reply:
Will switch to using `getResourceAsStream` on next commit.